src/shims/manual/mscorlib.csproj

```diff
@@ -4,36 +4,83 @@
-    <SeedTypePreference Include="System.Reflection.Emit.ModuleBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Reflection.Emit.ParameterBuilder" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.PropertyBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Reflection.Emit.SignatureHelper" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.TypeBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.DefaultInterfaceAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.EventRegistrationToken" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.EventRegistrationTokenTable`1" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.IActivationFactory" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.InterfaceImplementedInVersionAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.ReadOnlyArrayAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.ReturnValueNameAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.WindowsRuntimeMarshal" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.WriteOnlyArrayAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
+    <Compile Include="mscorlib.forwards.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsAot)' == 'true' or '$(TargetsUap)' == 'true'">
+    <ReferencePath Include="$(_RuntimePath)System.Reflection.Emit.dll" Aliases="System_Reflection_Emit" />
+    <ReferencePath Include="$(_RuntimePath)System.Reflection.Emit.Lightweight.dll" Aliases="System_Reflection_Emit_Lightweight" />
+    <ReferencePath Include="$(_RuntimePath)System.Reflection.Emit.ILGeneration.dll" Aliases="System_Reflection_Emit_ILGeneration" />
+    <ReferencePath Include="$(_RuntimePath)System.Runtime.InteropServices.WindowsRuntime.dll" Aliases="System_Runtime_InteropServices_WindowsRuntime" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <ReferencePath Include="$(_RuntimePath)Microsoft.Win32.Registry.dll" Aliases="Microsoft_Win32_Registry" />
```

Comment:
We should have a comment about this working around the fact that 2.0 s.p.c had internalsvisibleto mscorlib and included its own copy of these types.

Reply:
Why do we still care about 2.0 s.p.c in current master?

Reply:
This is here because we are still adding new types in netstandard2.0-targeting assemblies that overlap with desktop surface area.  This builds a new set of compat shims that make use of those (assuming we weren't planning to deliver fixes already).  /cc @safern

Reply:
I talked to @safern: we are only building these for netcoreapp2.0 and not delivering updates at all for netcoreapp2.1.  In that sense we can just delete this.  That will also save quite a bit of headache for @Anipik.

Reply:
https://github.com/dotnet/corefx/pull/36644